lib/modules/announcements.js

```diff
@@ -31,6 +31,8 @@ const recheckPostAfter = HOUR;
 
 const now = Date.now();
 
+const currentBrowser = BrowserDetect.browser;
```

Comment:
Could just check `${BrowserDetect.browser}` below instead of `${currentBrowser}`, doesn't seem necessary to alias it for a single use..